src/io/output_mesh.hh

```diff
@@ -132,6 +132,13 @@ public:
 		else return shared_from_this();
 	};
 
+	/// Return local element index to its global index.
+	inline LongIdx get_loc_elem_idx(LongIdx global_idx) {
```

Comment:
Is just "temporary" solution. comment where it is used, and that this should not be used elsewhere.